src/v2/Components/NavBar/Menus/NotificationsMenu.tsx

```diff
@@ -42,87 +76,103 @@ export const NotificationMenuItems: React.FC<NotificationsMenuQueryResponse> = p
-      </Flex>
+      {notifications.map(
+        ({ node: { artists, href, image, summary } }, index) => {
+          return (
+            <MenuItem
+              href={`${href}/works-for-sale`}
+              key={index}
+              aria-label={`${summary} by ${artists}`}
+              onClick={() => {
+                handleClick(href, AnalyticsSchema.Subject.Notification)
+              }}
+            >
+              <NotificationMenuItemContent
+                aria-hidden="true"
+                image={
+                  <Image
+                    src={image.thumb.url}
+                    width={40}
+                    height={40}
+                    alt={summary}
+                  />
+                }
+                summary={<Text variant="small">{summary}</Text>}
+                artists={
+                  <Text variant="small" fontWeight="bold">
+                    {artists}
+                  </Text>
+                }
+              />
+            </MenuItem>
+          )
+        }
+      )}
+
+      {notifications.length === 0 && (
+        <MenuItemPlaceholder justifyContent="center">
+          <Text variant="small">No new works</Text>
+        </MenuItemPlaceholder>
+      )}
+
+      <Box px={2}>
+        <Separator />
+      </Box>
+
+      <Link
+        display="block"
+        aria-label="View all notifications"
+        href="/works-for-you"
+        onClick={() => {
+          handleClick("/works-for-you", AnalyticsSchema.Subject.ViewAll)
+        }}
+      >
+        <Text variant="text" textAlign="center" p={2}>
+          View all
+        </Text>
+      </Link>
     </>
   )
 }
 
 /**
  * The <Menu /> component renders a QueryRenderer inside of it, which fetches
- * individual MenuItems for display. During fetch there is a loading spinner.
+ * individual MenuItems for display. During fetch there is a custom placeholder
  */
 export const NotificationsMenu: React.FC = () => {
   return (
-    <Menu title="Activity">
+    <Menu title="Activity" py={0} pt={1} aria-live="assertive">
       <NotificationsQueryRenderer
-        render={renderWithLoadProgress(
-          NotificationMenuItems,
-          {},
-          {},
-          { size: "small" }
-        )}
+        render={({ error, props }) => {
+          if (error) {
+            return (
+              <MenuItemPlaceholder justifyContent="center">
+                <Text variant="small" color="red100">
+                  {error.message}
+                </Text>
+              </MenuItemPlaceholder>
+            )
+          }
+
+          if (!props) {
+            return (
+              <>
+                <NotificationMenuItemPlaceholder />
+                <NotificationMenuItemPlaceholder />
+
+                <Box px={2}>
+                  <Separator />
+                </Box>
+
+                <SkeletonText variant="text" textAlign="center" p={2}>
+                  View all
+                </SkeletonText>
+              </>
```

Comment:
Might be a bit clearer in the future if this block was moved into a `<LoadingState />` component